src/Layout.js

```diff
@@ -0,0 +1,13 @@
+import ComponentRegistry from '../bower_components/metal/src/component/ComponentRegistry';
```

Comment:
Actually, this whole file is following an older convention, and also assuming older code. The new version won't work with this file in the end.

You should replace this file with the following contents:

```js
'use strict';

import templates from './Layout.soy.js';
import Component from 'metal-component';
import Soy from 'metal-soy';

class Layout extends Component {
}
Soy.register(Layout, templates);

export default Layout;
```